README.md

```diff
@@ -13,3 +13,38 @@ Add `.eslintrc` file:
   "extends": "@apify"
 }
 ```
+
+Create `tsconfig.json` file in the root of the project and add:
+
+```json
+{
+    "compilerOptions": {
+        "target": "es5",
+        "lib": [
+            "dom",
+            "dom.iterable",
+            "esnext"
+        ],
+        "allowJs": true,
+        "skipLibCheck": true,
+        "strict": true,
+        "forceConsistentCasingInFileNames": true,
+        "noEmit": true,
+        "esModuleInterop": true,
+        "module": "esnext",
+        "moduleResolution": "node",
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "jsx": "preserve",
+        "noUnusedLocals": true,
+        "noUnusedParameters": true
+    },
+    "exclude": [
+        "node_modules",
+    ],
+    "include": [
+        "**/*.ts",
+        "**/*.tsx"
+    ]
+}
+```
```

Comment:
This is very project specific. Do we need all those options? I assume people who use TS will already have their TS file so they only need to add the ones that are relevant to ESLint, no?

Reply:
Yes you are right tsconfig is project specific. I updated readme with a minimal example setup that works for web